docs/install.md

```diff
@@ -1,717 +1,132 @@
 
-* [Configuring PipelineResource storage](#configuring-pipelineresource-storage) to set up artifact storage for Tekton Pipelines.
-* [Customizing basic execution parameters](#customizing-basic-execution-parameters) if you need to customize your service account, timeout, or Pod template values.
+   - **Specific release:**
 
-### Installing Tekton Pipelines on OpenShift
+     ```bash
+      kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/previous/<version_number>/release.yaml
+     ```
 
-To install Tekton Pipelines on OpenShift, you must first apply the `anyuid` security
-context constraint to the `tekton-pipelines-controller` service account. This is required to run the webhook Pod.
-See
-[Security Context Constraints](https://docs.openshift.com/container-platform/4.3/authentication/managing-security-context-constraints.html)
-for more information.
+     Replace `<version_number>` with the numbered version you want to install.
+     For example, `v0.26.0`.
 
-1. Log on as a user with `cluster-admin` privileges. The following example
-   uses the default `system:admin` user:
+   - **Untagged release:**
 
-   ```bash
-   oc login -u system:admin
-   ```
-
-1. Set up the namespace (project) and configure the service account:
-
-   ```bash
-   oc new-project tekton-pipelines
-   oc adm policy add-scc-to-user anyuid -z tekton-pipelines-controller
-   oc adm policy add-scc-to-user anyuid -z tekton-pipelines-webhook
-   ```
-1. Install Tekton Pipelines:
+     If your container runtime does not support `image-reference:tag@digest`:
 
-   ```bash
-   oc apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.notags.yaml
-   ```
-   See the
-   [OpenShift CLI documentation](https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_cli/getting-started-cli.html)
-   for more information on the `oc` command.
+     ```bash
+     kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.notags.yaml
+     ```
 
-1. Monitor the installation using the following command until all components show a `Running` status:
+1. Monitor the installation:
 
    ```bash
-   oc get pods --namespace tekton-pipelines --watch
+   kubectl get pods --namespace tekton-pipelines --watch
    ```
 
-   **Note:** Hit CTRL + C to stop monitoring.
-
-Congratulations! You have successfully installed Tekton Pipelines on your OpenShift environment. Next, see the following topics:
-
-* [Configuring PipelineResource storage](#configuring-pipelineresource-storage) to set up artifact storage for Tekton Pipelines.
-* [Customizing basic execution parameters](#customizing-basic-execution-parameters) if you need to customize your service account, timeout, or Pod template values.
-
-If you want to run OpenShift 4.x on your laptop (or desktop), you
-should take a look at [Red Hat CodeReady Containers](https://github.com/code-ready/crc).
-
-## Configuring PipelineResource storage
```

Comment:
It looks like this block is being removed, but I'm not sure whether we should remove PipelineResources docs yet even though they are deprecated. Maybe we could just move to a less prominent location?

Reply:
I moved that Openshift note to the Openshift tab, you can find it at the bottom of that tab here: https://deploy-preview-469--tekton.netlify.app/docs/installation/pipelines/

As for the PipelineResources storage section, I just added it back in the post-installation configurations.

Thank you!